.github/workflows/sam_release_code.yml

```diff
@@ -156,3 +156,44 @@ jobs:
           # TRUSTSTORE_FILE: ${{ inputs.TRUSTSTORE_FILE }}
           VERSION_NUMBER: ${{ inputs.VERSION_NUMBER }}
         run: ./release_code.sh
+
+      - name: Set ECR Repository
+        run: echo "ECR_REPOSITORY=fhir-facade-repo" >> "$GITHUB_ENV"
+
+      - name: Determine Image Tag
+        id: determine-image-tag
+        run: |
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
```

Comment:
see other comment about getting the tag. This file already has inputs of version_number and commit_id so you can use them rather than trying to calculate it